### PR TITLE
Fix bug Issue4592

### PR DESCRIPTION
**Pull request by JiahangXu**
### Description ###

fix bug in issue #4592 

### Checklist ###
  - [ ] test case
  - [ ] doc

### CODE REVIEW

File: examples/nas/multi-trial/mnasnet/base_mnasnet.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 import torch
 import torch.nn as torch_nn
-from torchvision.models.utils import load_state_dict_from_url
 import torch.nn.functional as F
+from nni.retiarii.serializer import model_wrapper
```
Comment (ultmaster) [inline examples/nas/multi-trial/mnasnet/base_mnasnet.py:8]:
```python
from nni.retiarii import model_wrapper
```

File: examples/nas/multi-trial/mnasnet/base_mnasnet.py [hunk 2]
```diff
@@ -180,7 +180,7 @@ def __init__(self, alpha, depths, convops, kernel_sizes, num_layers,
             nn.ReLU(inplace=True),
         ]
         self.layers = nn.Sequential(*layers)
-        self.classifier = nn.Sequential(nn.Dropout(p=dropout, inplace=True),
```
Comment (QuanluZhang) [inline examples/nas/multi-trial/mnasnet/base_mnasnet.py:183]: why adding `inplace=True` induces problem?

Comment (JiahangXu) [reply examples/nas/multi-trial/mnasnet/base_mnasnet.py:183]: Error info says 'one of the variables needed for gradient computation has been modified by an inplace operation.'
The new version of PyTorch does not support inplace operation, so that the back propagation is interrupted by the `inplace` operation in dropout layer. Here is a similar issue in pytorch: https://discuss.pytorch.org/t/help-runtimeerror-one-of-the-variables-needed-for-gradient-computation-has-been-modified-by-an-inplace-operation/107030/2.

Comment (QuanluZhang) [reply examples/nas/multi-trial/mnasnet/base_mnasnet.py:183]: so this is an issue of pytorch not nni, right?

**Review by QuanluZhang — APPROVED** [review]

**Review by ultmaster — APPROVED** [review]
